orga/tests/integration/components/sco-organization-participant/list_test.js

```diff
@@ -92,6 +91,61 @@ module('Integration | Component | ScoOrganizationParticipant::List', function (h
       .exists();
   });
 
+  test('it should display participant as eligible for certification', async function (assert) {
```

Comment:
**suggestion :** A mon avis on peut donner un peut de contexte aus tests.
'it should display participant as eligible for certification when the participant is certifiable'
'it should display participant as non eligible for certification when the participant is not certifiable'
it should display that no information about eligibility for certification are available when when there is no informations'

Reply:
j'ai déplacé les tests dans le fichier du nouveau composant, du coup je garde 1 seul test sur "est-ce qu'on file bien ce qu'il faut"

Reply:
d'acc !